common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -623,6 +623,11 @@ class @Problem
               return false
           }
         }
+      id = element.attr('id').replace(/^input_/, '')
+      CodeMirrorTextArea = CodeMirrorEditor.getInputField()
+      CodeMirrorTextArea.setAttribute('id', 'cm-textarea-' + id)
+      CodeMirrorTextArea.setAttribute('aria-describedby', 'status_' + id + ' ' + 'cm-editor-exit-message_' + id)
```

Comment:
My only advice is that we reverse the order of these describedbys. The message to exit should come first and then the state of the question.